corehq/apps/auditcare/models.py

```diff
@@ -165,6 +166,10 @@ class AccessAudit(AuditEvent):
     http_accept_fk = models.ForeignKey(
         HttpAccept, null=True, db_index=False, on_delete=models.PROTECT)
     http_accept = ForeignValue(http_accept_fk, truncate=True)
+    trace_id = models.CharField(max_length=127, null=True, blank=True)
```

Comment:
Null means "unknown", empty string means the string had no value. I would prefer every field to be `NOT NULL`, but that would require a much more significant refactor and make this PR much bigger. This specific field might not need to be null, but then it would probably need to have `default=""` and all places where it is set would need to have `None` converted to empty string, which seemed to add more complexity than it was worth.